tests/hotplug/cpu.go

```diff
@@ -148,7 +150,7 @@ var _ = Describe("[sig-compute][Serial]CPU Hotplug", decorators.SigCompute, deco
 				}
 				return false
 			}, 30*time.Second, time.Second).Should(BeTrue())
-			tests.ExpectMigrationSuccess(virtClient, migration, tests.MigrationWaitTime)
+			libmigration.ExpectMigrationToSucceed(virtClient, migration, libmigration.MigrationWaitTime)
```

Comment:
nit: maybe add another method inside `libmigration` that avoids repeating the `libmigration.MigrationWaitTime` in this file three times?

```suggestion
			libmigration.ExpectMigrationToSucceedWithDefaultTimeout(virtClient, migration)
```